### DISCUSSION TITLE
Multi peer addresses - introduced in ZeroTier v1.8.5+

### DISCUSSION THREAD

**Discussion by tinola** [key-networks/ztncui#88] [General]
Hi,

In ZeroTier v1.8.5 ztncui started display multi peer connections addresses in the last members column.
The same occurs in ZT v.1.8.6.

Because I don't know completely PUG coding, I found **very very dirty** workaround on line:

https://github.com/key-networks/ztncui/blob/8fb29fa60771bdaa1e66157c7c383cb56afa0c8f/src/views/network_detail.pug#L137

I changed to:

`each path in peer.paths.slice(0, 1)`

Could anyone fix it in correct way? :)

**Comment by lideming** [comment]
I have made some changes in my repo, showing the most recent path only:
https://github.com/lideming/ztncui/commit/671525e331791c32709c6a23a78c952358ca5d5e

**Comment by tinola** [comment]
Thank you @lideming, just cloned your "wip" branch, and members list looks much much better :)
